Python/bytecodes.c

```diff
@@ -4096,9 +4554,9 @@ dummy_func(
         }
 
         inst(INSTRUMENTED_POP_JUMP_IF_TRUE, (unused/1 -- )) {
-            PyObject *cond = POP();
-            assert(PyBool_Check(cond));
-            int flag = Py_IsTrue(cond);
+            _PyStackRef cond = POP();
+            assert(PyBool_Check(PyStackRef_AsPyObjectBorrow(cond)));
+            int flag = PyStackRef_IsTrue(cond);
```

Comment:
```suggestion
            int flag = PyStackRef_Is(cond, PyStackRef_True);
```
And the same for the similar instructions below